### PR TITLE
[CSL-604] Create catalog endpoints

### PR DESCRIPTION
**Pull request by esezen**

### CODE REVIEW

File: constructor_io/modules/catalog.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        if items:
+            file_data['items'] = ('items.csv', items)
+
+        if variations:
+            file_data['variations'] = ('variations.csv', variations)
+
+        if item_groups:
+            file_data['item_groups'] = ('item_groups.csv', item_groups)
+
+
```
Comment (crgee1) [inline constructor_io/modules/catalog.py:45]: I don't think this extra line is the norm, but maybe

File: tests/modules/test_catalog.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
+
```
Comment (crgee1) [inline tests/modules/test_catalog.py:1]: Can we test for task id is returned like for the node tests?

**Review by crgee1 — CHANGES_REQUESTED** [review]
Runs great, nice work, just some updates to the tests

File: constructor_io/modules/catalog.py (hunk 1 above)
Comment (esezen) [reply constructor_io/modules/catalog.py]: It is not. Good catch

File: tests/modules/test_catalog.py (hunk 2 above)
Comment (esezen) [reply tests/modules/test_catalog.py:1]: Just added those. Didn't realize we are checking those in node
Good catch!

**Review by crgee1 — APPROVED** [review]
Looks good, not necessary and I forgot to check last time but maybe some tests for the for `force` and `notification_email` and a red line test for `path` to get the coverage percent up (it's at 91% so it's not like it's bad)

**Review by crgee1 — APPROVED** [review]
lgtm

**Pull request merged**